src/gdb_packet.c

```diff
@@ -31,121 +31,163 @@
 
 #include <stdarg.h>
 
+typedef enum packet_state_e {
```

Comment:
the _e on the end here is wrong (but correct below)

Reply:
ups, that was the find and replace